nni/common/concrete_trace_utils/concrete_tracer.py

```diff
@@ -546,7 +560,17 @@ def trace(self, root: Union[torch.nn.Module, Callable[..., Any]],
             fn = root
 
         tracer_cls = getattr(self, '__class__', None)
-        self.graph = Graph(tracer_cls=tracer_cls)
+        self.graph = Graph(tracer_cls=tracer_cls, tracer_extras={
```

Comment:
I find that `tracer_extras` is not existed in my `torch.fx`...

Reply:
![image](https://user-images.githubusercontent.com/77614475/222428878-bab579a5-f5c9-46fe-96fb-9f70bb378eae.png)
In [pytorch fx document](https://pytorch.org/docs/1.13/fx.html#api-reference), here introduces that the fx.Graph has this parameter.

Reply:
Right, the `tracer_extras` is useless. I deleted it and please check the latest commit to see if there are any problems.